shared/weapons.lua

```diff
@@ -1,160 +0,0 @@
----@class Weapon
----@field name string
----@field label string
----@field weapontype string
----@field ammotype? string
----@field damagereason string
-
----@deprecated This file is deprecated and will be removed in the future. If you are utilizing QB-Core bridge functionality you will need to populate weapons here for them to be available in QBCore.Shared.Weapons. If not please add your weapons in ox_inventory/data/weapons.lua file. Currently weapons in this file will be converted on next server start.
----@type table<number, Weapon>
-return {
-	-- // WEAPONS
-	-- Melee
-	[`weapon_unarmed`] 				 = {name = 'weapon_unarmed', 		label = 'Fists', 				weapontype = 'Melee',	ammotype = nil, damagereason = 'Melee killed / Whacked / Executed / Beat down / Murdered / Battered'},
```

Comment:
Since ox_inventory format is missing weapontype, ammotype, and damagereason, we need to keep those fields in this file. They can then be combined with the ox_inventory data to recreate the format of QB weapons. Otherwise this will break compatibility with scripts that rely on these fields.